ui-react/packages/atlasmap-provider/src/useAtlasmap.tsx

```diff
@@ -138,6 +138,13 @@ export function useAtlasmap({
     };
   }, [initializationService]);
 
+  const handleExportAtlasFile = useCallback(
+    (file: File) => {
+      exportAtlasFile(file);
+    },
+    [dispatch]
```

Comment:
We don't need a `useCallback` in this case since the `exportAtlasFile` callback doesn't depend on anything created in this component.